bin/wasm-node/javascript/src/index.ts

```diff
@@ -14,6 +14,4 @@
 
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-export { HealthChecker, SmoldotHealth, healthChecker } from './health.js';
 export * from './client.js'; // TODO: precise exports
```

Comment:
```suggestion

export * from './client.js'; // TODO: precise exports
```

Very nit picky but I'd like a new line please